### DISCUSSION TITLE
Based on #10758, Multi-site entries in multi-site structures can't be ordered independently per-site

### DISCUSSION THREAD

**Discussion by kevinmu17** [craftcms/cms#13873] [Ideas] [answered]
As stated in #10758 by @brandonkelly this is by design. But when you setup your section with the Propagation Method to `Let each entry choose which sites it should be saved to` this seems not a desirable design. 

You may want to change the order per site. This was my expectation of how this setting would work when ordering them entries.

In my case, i have 4 entries that share a lot of structure, but the possibility to change the order between them on different sites (priority per entry is different for each site) is desirable. So now i have to change the Propagation Method to `Only save entries to the site they were created in` And tell my client to add & fill everything twice?

**Comment by brandonkelly** [comment, marked as answer]
> As stated in #10758 by @brandonkelly this is by design. But when you setup your section with the Propagation Method to `Let each entry choose which sites it should be saved to` this seems not a desirable design.

It might not be for your specific use case, but in general it appears to be the desired behavior, given that this is the first time this suggestion has come up.

And if it’s generally working as expected, we should not change it, because per-site structures would add a significant amount of additional legwork to the authoring workflow. You’d have to start choosing where an entry is positioned for each site, every time you create a new entry or change its position.

(And I don’t think we should give the “Let each entry choose…” propagation method any sort of special treatment here. Most structures that use that propagation method have a fair bit of crossover between the sites.)

> So now i have to change the Propagation Method to `Only save entries to the site they were created in` And tell my client to add & fill everything twice?

If it’s only four entries, then maybe that’s the best option.

If you’re only using the structure for sorting, but not for multi-level hierarchy, you could also consider other ways of sorting the entries, such as by a relational field that lives on a Single entry, or a “Sort Order” custom field that entries get ordered by.
